_pages/dev/global-configuration-in-spartacus.md

```diff
@@ -17,30 +15,29 @@ Any configuration provided can be overridden in the main app module.
 
 ***
 
-## How to use configuration
+## Providing Global Configurations in Spartacus
+
+The following sections describe the different ways you can provide global configuration in Spartacus.
 
 ### StorefrontLib.withConfig
 
-If you import B2cStorefrontModule in your app, preferred and easiest method to provide configuration is to use `B2cStorefrontModule.withConfig(config?: StorefrontModuleConfig)`.
-`StorefrontModuleConfig` interface provides type safety and context-aware code completion that speed ups defining configuration and helps to avoid any typos.
+If you import the `B2cStorefrontModule` in your app, the preferred and easiest method to provide a configuration is to use `B2cStorefrontModule.withConfig(config?: StorefrontModuleConfig)`. The `StorefrontModuleConfig` interface provides type safety and context-aware code completion that speeds up defining the configuration and helps to avoid any typos.
 
 ### ConfigModule.withConfig
 
-Importing `ConfigModule.withConfig(config: any)` is useful, when we want to use the configuration in our module and, at the same time, contribute to it.
-`config` parameter is not typed, can be an object of any shape, so it suggested to either use typed const or use type casting `ConfigModule.withConfig(<ConfigType>{})` to utilize type safety.
+Importing `ConfigModule.withConfig(config: any)` is useful when you want to use the configuration in your module and contribute to it at the same time. The `config` parameter is not typed and can be an object of any shape, so it is recommended that you use either a typed `const`, or that you use `ConfigModule.withConfig(<ConfigType>{})` type casting to be able to use type safety.
 
 ### provideConfig
```

Comment:
ProvideConfig should be used as preffered one, `ConfigModule.withConfig` as the legacy one, and `(B2c)StorefrontModule.withConfig` are dropped already.

Reply:
I have changed the order of the sections to reflect your comment, I have pretty much removed the contents of the `StorefrontModule.withConfig` section (except to say that it is no longer supported), and I have added a line at the beginning of the other sections to say which one is the preferred method and which one is the legacy method.

Beyond that, I assume we should modify what the text currently says about `provideConfig` and `ConfigModule.withConfig`. Do you have any suggestions? Or is there somone I can follow up with to ask for a "rewrite" of these sections? 🙂

Reply:
👍